apps/src/lab2/lab2Redux.ts

```diff
@@ -40,6 +40,13 @@ export interface LabState {
   isLoadingProjectOrLevel: boolean;
   // If the lab is loading. Can be updated by lab-specific components.
   isLoading: boolean;
+  // Error currently on the page, if present. If not undefined, isPageError should also be true.
+  pageError:
```

Comment:
could we derive `isPageError` from `pageError !== undefined` (i.e. have a selector instead of two pieces of the redux store)?

Reply:
Oh yep good call, will do